provisioners/configure-backend.sh

```diff
@@ -0,0 +1,4 @@
+./configure.sh
```

Comment:
I think we need the [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) here.

```suggestion
#!/usr/bin/env bash

./configure.sh
```